.github/workflows/ci_cd.yml

```diff
@@ -1182,6 +1182,22 @@ jobs:
           wait_time: 60 # check every minute
           max_time: 1800 # allow up to 30 minutes for a deployment
 
+      - name: Setup k6
+        uses: grafana/setup-k6-action@v1
+
+      - name: Login to k6 cloud
+        env:
+          K6_CLOUD_TOKEN: ${{ secrets.GC_K6_TOKEN }}
+        run: k6 login cloud --token "$K6_CLOUD_TOKEN"
```

Comment:
According to the docs the mere presence of a `K6_CLOUD_TOKEN` environment variable is sufficient for authentication:

https://grafana.com/docs/grafana-cloud/testing/k6/author-run/tokens-and-cli-authentication/#authenticate-with-an-environment-variable

So, the login step could be removed and the env block here could be merged with that of the following "Run k6 script" step.

Reply:
Ah! I thought that may be the case but didn't look hard enough in the docs! I think I was looking at the k6 docs, rather than Grafana Cloud k6 docs (lesson to be learned there).

Great call, thanks for checking it out.